src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="Search"
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+
+    >
+      <img src="images/Search.svg"
+           alt="Search"
+           class="imgSearch"
+      >
+      <input
+        class="input"
+        type="search"
+        id="Search/big"
+        name="Search"
```

Comment:
Usually name comes in camelCase - so the name should not start from uppercase letter
```suggestion
        name="search"
```